src/content/glossary/index.md

```diff
@@ -530,6 +530,10 @@ A network [node](#node) that finds valid [proof-of-work](#pow) for new blocks, b
   Mining
 </DocLink>
 
+### Mint {#mint}
+
+Minting is the process of creating new coins and bringing them into circulation so that they can be traded. It's a decentralised mechanism to create a new token without the involvement of the central authority.
```

Comment:
```suggestion
Minting is the process of creating new tokens and bringing them into circulation so that they can be traded. It's a decentralised mechanism to create a new token without the involvement of a central authority.
```

Reply:
@samajammin any thoughts on saying coins/tokens here (or both). I think token catches everything and coins creates a weird mental model.

Reply:
Also, is it important that we mention trading tokens here? Is that their purpose? Not sure myself... many tokens aren't explicitly designed to have trading value (e.g. governance tokens)

Reply:
@minimalsm I think we should use both coins and tokens.

Reply:
Not a string opinion but I think "token" makes sense, as minting is most relates to token contracts (i.e. ERC-20 or ERC-721 contracts).

Reply:
I don't have a very strong opinion here either, but I honestly feel "coin" is the more awkward option given how we often talk about minting tokens of NFTs.

I think given the context it's simplest and makes most sense to just go with "token." It fits the idea of the "token contracts" we're familiar with (not "coin contracts")

```suggestion
Minting is the process of creating new tokens and bringing them into circulation so that they can be traded. It's a decentralized mechanism to create a new token without the involvement of the central authority.
```
(Also, small American English tweak per style guide)